indico/web/views.py

```diff
@@ -142,6 +143,8 @@ def _getPageContent(self, params):
         if html is not None:
             return html
         template = params.pop('_jinja_template')
+        webpack = current_app.extensions['flask-webpackext']
+        params['bundles'] = map(lambda x: webpack.manifest[x], self.bundles)
```

Comment:
I think the BDFL hates `map(lambda: ...)` in favor of list comprehensions ;)

`map(webpack.manifest.__getitem__, self.bundles)` would work but I think it's uglier than just using a list comprehension

Reply:
Yeah, that's the problem. I initially did it like that and then moved to the `lambda` 😉